lib/uplink/apikey.go

```diff
@@ -17,6 +17,15 @@ func (a APIKey) Serialize() string {
 	return a.key.Serialize()
 }
 
+func (a APIKey) serializeRaw() []byte {
+	return a.key.SerializeRaw()
+}
+
+// IsZero returns if the api key is an uninitialized value
+func (a *APIKey) IsZero() bool {
+	return a.key == nil
+}
```

Comment:
Is this a necessary API to export? Can't you do `key == (APIKey{})`? On the other hand, `IsZero` matches `time.Time`, and allows us to add incomparable fields in the future.